packages/yoastseo/spec/specHelpers/scoring/seoAssessorTests.js

```diff
@@ -89,7 +90,7 @@ export function checkAssessmentAvailability( assessor, isProductAssessor = false
 		const paper = new Paper( "", { keyword: "a" } );
 		const assessments = assess( paper );
 
-		expect( assessments.sort() ).toEqual( defaultAssessments.concat( "functionWordsInKeyphrase" ).sort() );
+		expect( assessments.sort() ).toEqual( textAssessments.concat( "functionWordsInKeyphrase" ).sort() );
 	} );
 
 	it( "additionally runs assessments that require text and a keyword", function() {
```

Comment:
Unless I'm missing something, this seems to be a duplicate of the test on line 82.

Reply:
Correct, removed! 👍